staging/src/k8s.io/code-generator/kube_codegen.sh

```diff
@@ -538,7 +538,7 @@ function kube::codegen::gen_client() {
     done < <(
         ( kube::codegen::internal::git_grep -l \
             -e '+genclient' \
-            ":(glob)${in_root}"/'**/types.go' \
+            ":(glob)${in_root}"/'**/*types.go' \
```

Comment:
Can you also include files with names as `types_*` ?
```suggestion
            ":(glob)${in_root}"/'**/*types*.go' \
```